test/pthread/test_pthread_unhandledrejection.c

```diff
@@ -1,6 +1,6 @@
 #include <emscripten/emscripten.h>
 
 int main() {
-  EM_ASM({ Promise.reject("rejected!"); });
+  EM_ASM({ Promise.reject(new Error("rejected!")); });
```

Comment:
(Node transforms uncaught promise rejections into Error objects, but Deno and Bun don't do this. The test this is used in would then fail in Deno and Bun because it expected to get an Error object. To fix the test, I just made it always create an Error object. This did require me to slightly update the expected output in `test/test_browser.py` for the browser versions of this test.)